components/self-service/new-content-form.tsx

```diff
@@ -116,7 +116,12 @@ export default function NewContentForm({ onSubmit, defaultValues }: { onSubmit?:
       setValue("deliveryRepoApplicationCategories", new Set(newCategories));
       setSelectedAppCategories(newCategories);
     }
-};
+  };
+
+  const [userImageReleaseVersion, setUserImageReleaseVersion] = useState('openshift-4.17')
```

Comment:
Can we dynamically get the highest branch on `ocp-build-data`, like `openshift-4.19` at this point of time?

Reply:
this is now implemented